releasenotes/notes/channel-module-deprecations-in-pulse-ffc384b325b077f1.yaml

```diff
@@ -0,0 +1,23 @@
+---
+upgrade:
+  - |
+    Channel ``buffer`` option was deprecated 4 months ago and has now been removed.
```

Comment:
Oh, I should have noticed on the last review. 4 months isn't actually enough context for the release notes. The 0.13.0 release could be another 4 months away. We should say the version it was deprecated in instead because that's a fixed point in the past.